.travis.yml

```diff
@@ -40,6 +40,25 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update conda
 
+<<<<<<< HEAD
```

Comment:
Conflict marker?

Reply:
👍 , that might be why its trying to do a ruby build